onnx/defs/tensor/defs.cc

```diff
@@ -1848,4 +1848,52 @@ ONNX_OPERATOR_SET_SCHEMA(
 
           propagateShapeFromInputToOutput(ctx, 0, 0);
         }));
+
+static const char* Unique_ver11_doc = R"DOC(
+Finds unique elements in an input 1-D tensor sorted in the same order of first occurrence in the input.
```

Comment:
not sure it's because of my personal understanding of English. Is it better to say dedup? My first thought of unique of input {2, 1, 1, 3, 4, 3} is {2, 4} :)

Reply:
This is good feedback. I will try and refine the description - but I was going based on the tf.unique description - 

`This operation returns a tensor y containing all of the unique elements of x sorted in the same order that they occur in x. This operation also returns a tensor idx the same size as x that contains the index of each value of x in the unique output y` from https://www.tensorflow.org/api_docs/python/tf/unique